src/styles/modal-component.less

```
@@ -54,6 +54,10 @@
     position: absolute;
   }
 }
+.available-volunteers{
+  background-color: #d5d3d1;
```

Lagre fargevariabler i colors.less og heller skriv:
```suggestion
  background-color: @volounteer-color;
```

eller noe i den dur

---

Er dette gjort?